src/client/components/series-menu/format-picker.tsx

```diff
@@ -29,7 +29,7 @@ interface FormatPickerProps {
   formatChange: Unary<SeriesFormat, void>;
 }
 
-function readFormat(format: string, measureFormat: string): SeriesFormat {
+function readFormat(format: string, measureFormat: {} | string): SeriesFormat {
```

Comment:
Use `FormatString` type instead of inlining it. I know that numbro types are quite poor, but let's cover them with good names.